docs/Policies/ConflictResolutionPolicy.mdx

```diff
@@ -15,16 +15,79 @@ this should be same as sidebar_label and folder name
 
 should be present on every controlled document.
```

Comment:
@VictorAMartinez  I think you missed removing this line.
and also line no 12 "this should be same as sidebar_label and folder name"